pages/design/atomic-template-empty.md

```diff
@@ -1,35 +1,38 @@
 ---
+id: 
 name:
 rule_type: atomic
 
 description: |
   This rule checks ...
 
-success_criterion:
-  - x.x.x # (Name of Success Criterion)
+accessibility_requirements:
+  wcag20:x.x.x: # (Name of Success Criterion)
```

Comment:
Might be needed: Instructions for what to do if no accessibility requirements exist.

https://github.com/act-rules/act-rules.github.io/pull/500#issuecomment-489584544

Reply:
I agree! Especially when the rule is part of a composite rule.